tests/playwright-tests/general-public/pages/FindStatisticsPage.ts

```diff
@@ -11,7 +11,7 @@ export default class FindStatisticsPage {
   }
 
   async navigateToPublicReleasePage(publicationName: string) {
-    await this.releaseLink(publicationName).click();
-    await this.page.waitForTimeout(2000);
+    await this.releaseLink(publicationName).click({ force: true });
+    await this.page.waitForNavigation();
```

Comment:
Nice, looks like waiting for all the navigation events to occur is a good way to go! Looks like this method is deprecated though, the docs say to use `waitForURL` instead -> https://playwright.dev/docs/api/class-page#page-wait-for-navigation